src/django_mysql/models/fields/fixedchar.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from typing import Any, cast
+
+from django.db.backends.base.base import BaseDatabaseWrapper
+from django.db.models import CharField
+
+from django_mysql.typing import DeconstructResult
+
+
+class FixedCharField(CharField):
+    def __init__(self, *args: Any, length: int, **kwargs: Any) -> None:
+        if not isinstance(length, int):
+            raise TypeError(
+                'Invalid length value "{length}". '
+                "Expected integer value.".format(length=length)
+            )
```

Comment:
This check is unnecessary - we already have both the type hint and the system check in `CharField`:

https://github.com/django/django/blob/7c4f3965098baad2396e24501e09237425a7bd6f/django/db/models/fields/__init__.py#L1020